### ISSUE TITLE
anzu-query-replace-regexp: match-substitute-replacement: Invalid use of ‘\’ in replacement text

### ISSUE THREAD

**Issue by sshaw** [emacsorphanage/anzu#112]
Hi, thanks for taking over maintenance of this.

When a replacement contains `\,` is passed to `anzu-query-replace-regexp` the following error occurs:

    match-substitute-replacement: Invalid use of ‘\’ in replacement text

For example:

```
be_\(truthy\|falsey\) -> be \,(if (string= \1 "truthy") "true" "false")
```

This gets highlighted correctly:

<img width="154" alt="Screen Shot 2020-04-22 at 6 42 20 PM" src="https://user-images.githubusercontent.com/17570/80041066-71c12e00-84c9-11ea-8064-db4b0cef3004.png">

But the replacement fails with the above error. 

<kbd>M-x</kbd> `query-replace-regexp` works.

**Comment by syohex** [comment]
Did you use latest version ? I suppose this issue was fixed by #101. Please check latest version and try again.

**Comment by sshaw** [comment]
Yes, it was fixed by #101. Thanks!

@gonewest818 need a new release!

**Comment by gonewest818** [comment]
I just tagged release 0.63, you should see it in a few hours when the next MELPA stable build runs. Thanks!
